Source/nsimage-tiff.h

```diff
@@ -57,6 +57,9 @@ typedef struct {
     int     error;
     float   xdpi;
     float   ydpi;
+  short    isBigEndian;
```

Comment:
@fredkiefer is this a good way to hold BOOLs ?
This file is Objective-C agnostic, so no BOOL available.